scripts/performance.sh

```diff
@@ -14,9 +14,22 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+DEV=0
+
+for arg in "$@"
```

Comment:
Nit, since only `--dev` is supported, it might be easier to 
```suggestion
if [[ "$1" == "--dev" || "$1" == "-d" ]]; then
```